Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:24.04
+
+ARG USERNAME=hl
```

Comment:
how about `hluser` to be consistent with the readme?

Reply:
done